socorro/external/rabbitmq/crashstorage.py

```diff
@@ -170,7 +171,7 @@ def new_crashes(self):
         # queues the crash_id. The '_consume_acknowledgement_queue' function
         # is run to send acknowledgments back to RabbitMQ
         self._consume_acknowledgement_queue()
-        conn = self.rabbitmq.connection()
+        self.rabbitmq.connection()  # XXX is this ever used and needed?
```

Comment:
Can you not just remove that line? It looks pretty useless, since it doesn't return anything. That would be a major design error of the ``rabbitmq`` library if it was needed.